### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by S1rDan**
Fixing some issues the linter is coming up with.

### CODE REVIEW

**Review by mustiwebp — COMMENTED** [review]
PR is fine from my side!

File: lib/l10n/app_de.arb [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "husband": "Ehemann",
     "wife": "Ehefrau",
     "entryEmail": "Bitte geben Sie die E-Mail-Adresse ein!",
-    "entryValidEmail": "Btte Sie eine gültige E-Mail-Adresse ein!",
+    "entryValidEmail": "Bitte Sie eine gültige E-Mail-Adresse ein!",
```
Comment (inchw0rm) [inline lib/l10n/app_de.arb:56]: The message on line 56 is still not right

**Review by inchw0rm — CHANGES_REQUESTED** [review]

File: lib/l10n/app_de.arb (hunk 1 above)
Comment (inchw0rm) [reply lib/l10n/app_de.arb:56]: It's also fine from my side, please squash and merge when ready.

**Comment by S1rDan** [comment]
I've created the change. I can't merge that myself due to lack of rights.

**Pull request merged**